ush/eva/jedi_gsi_compare_conv.yaml

```diff
@@ -279,18 +279,20 @@ diagnostics:
             color: 'red'
             label: 'hofxdiff vs pressure'
           statistics:
-            data:
-              variable: experiment::HofXDiff::${variable}
-              @CHANNELKEY@
-            statistic list:
+            fields:
+              - field_name: experiment::HofXDiff::${variable}
+                @CHANNELKEY@
+                xloc: 0.5
+                yloc: -0.10
+                kwargs:
+                  fontsize: 6
+            statistics_variables:
             - n
             - min
             - mean
             - max
             - std
             - name
```

Comment:
this one still has `name` does that work?

Reply:
> this one still has `name` does that work?

@CoryMartin-NOAA Full disclosure, I was not able to test this YAML as none of the conventional YAMLs are giving valid output file right now.  So this is a bug.

I'm changing this to a draft PR until I find a way to test this one.

But yes, `name` needs to be removed.